rosdep/base.yaml

```diff
@@ -2513,6 +2513,9 @@ libsensors4-dev:
   debian: [libsensors4-dev]
   fedora: [lm_sensors-devel]
   ubuntu: [libsensors4-dev]
+libserial-dev:
+  debian: [libserial0, libserial-dev]
```

Comment:
It looks like libserial-dev has a dependency on libserial0. I'd prefer not to duplicate that dependency.